examples/supplementary/format_concat_args/.cargo/config.toml

```
@@ -0,0 +1,6 @@
+[target.'cfg(all())']
+rustflags = ["-C", "linker=dylint-link"]
+
+# For Rust versions 1.74.0 and onward, the following alternative can be used
+# (see https://github.com/rust-lang/cargo/pull/12535):
+# linker = "dylint-link"
```

```suggestion
target-dir = "../../target/examples"

[target.'cfg(all())']
linker = "dylint-link"
```
To match the other examples' configs.